app/src/js/modules/liveeditor.js

```diff
@@ -122,11 +128,22 @@
                                     win.getSelection().getRangeAt(0).insertNode(doc.createTextNode(content));
                                 }
                             }
-                        }).on('keypress', function (e) {
-                            return e.which != 13;
-                        }).on('focus blur', function (e) {
-                            $(this).html($(this).text());
                         });
+
+                        if(fieldType == 'textarea') {
+                            $(this).on('keypress', function (e) {
+                                if(e.which == 13) {
+                                    e.preventDefault();
+                                    doc.execCommand('insertHTML', false, '<br><br>');
+                                }
+                            });
+                        } else {
+                            $(this).on('keypress', function (e) {
+                                return e.which != 13;
+                            }).on('focus blur', function (e) {
+                                $(this).html($(this).text());
```

Comment:
The reason I did it differently was to ensure parity with the non-liveedit counterpart. In a textarea field you can have newlines and they will persist to storage. This allows things like outputting the field with `{{ content.textarea | nl2br }}` using `.text()` would mean you can have newlines when editing content via the panel and then they would disappear if you used live edit.

I wasn't aware of the "extra" contenteditable commands, browser vendors are terribly inconvenient. Seems like there are plans for a `contenteditable="plaintext"` functionality but nothing we could actually use yet.